### PR TITLE
Alvins backend branch

### PR DESCRIPTION
**Pull request by ajfrierson**
I have built out the users and admin Controllers/Repos and Models.

### CODE REVIEW

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/controllers/UsersController.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @RequestMapping(method = RequestMethod.POST)
+    public void postAdmin(Users user){
+        usersRepository.save(user);
+    }
+
+    // DELETE a user by ID
+    @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
+    public void deleteAdmin(Integer id){
+        usersRepository.deleteById(id);
+    }
```
Comment (jondgons) [inline travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/controllers/UsersController.java:47]: Should these methods refer to Users?

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Users.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        this.userName = userName;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
```
Comment (jondgons) [inline travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Users.java:77]: Do you want to try using Lombok's annotations? If not, no worries!

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Admin.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        this.userName = userName;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
```
Comment (jondgons) [inline travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Admin.java:77]: Do you want to try using Lombok's annotations? If not, no worries!

**Review by jondgons — CHANGES_REQUESTED** [review]

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/controllers/UsersController.java (hunk 1 above)
Comment (ajfrierson) [reply travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/controllers/UsersController.java:47]: I have refactored the code to refer to users.

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Users.java (hunk 2 above)
Comment (ajfrierson) [reply travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Users.java:77]: I am willing to try but am unfamiliar an might need you assistance.

File: travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Admin.java (hunk 3 above)
Comment (ajfrierson) [reply travel-planner/src/main/java/com/revature/team2/project2/travelplanner/beans/models/Admin.java:77]: We can definitely try.

**Review by jondgons — APPROVED** [review]

**Pull request merged**